vsutil/clips.py

```diff
@@ -229,3 +237,21 @@ def _should_dither(in_bits: int,
                 or float_to_int
                 or (in_range == types.Range.FULL and upsampling and (in_bits, out_bits) != (8, 16))
                 or downsampling)
+
+
+def shift_clip(clip: vs.VideoNode, offset: int) -> vs.VideoNode:
+    if offset > clip.num_frames - 1:
+        raise ValueError("shift_clip: Offset can't be greater than the clip length!")
+
+    if offset < 0:
+        return clip[0] * abs(offset) + clip[:offset]
+    elif offset > 0:
+        return clip[offset:] + clip[-1] * offset
+
+    return clip
+
+
+def shift_clip_multi(clip, shift: types.FrameRange = (-1, 1)):
```

Comment:
Missing return type annotation.

(The corresponding mypy option should be set to make incomplete function signatures fail the lint.)

Reply:
There is no linting in this package